package.json

```diff
@@ -585,6 +586,11 @@
                     "group": "0@2",
                     "when": "view == extension.vsKubernetesHelmRepoExplorer && viewItem =~ /vsKubernetes\\.((chart)|(chartversion))/i"
                 },
+                {
+                    "command": "extension.helmUninstall",
+                    "group": "3",
```

Comment:
was unsure here so have added a new group, happy to follow the conventions for groups if new group is not appropriate.